release-notes-6.0.0.md

```diff
@@ -4,6 +4,8 @@
 ${version-number}
 
 #### New Features
+- 614206: Diverted message checking can now be disabled using the CAF_WORKER_ENABLE_DIVERTED_TASK_CHECKING environment 
+variable.
```

Comment:
@andyreidz / @caoimhelynch Should duplicate this info on the new env variable in the [worker-document](https://github.com/CAFDataProcessing/worker-document/blob/develop/release-notes-5.2.0.md) release notes as well?

For example, the App Resources Worker pulls in the latest Worker Document version in order to get this new env var functionality from the Worker Framework:

https://github.houston.softwaregrp.net/Verity/darwin-worker-appresources/pull/106/files

Reply:
I'm ok with not duplicating it, they can find what they need by following through the documentation of document worker and worker framework.

Reply:
Thanks, looking at the [worker-document commits](https://github.com/CAFDataProcessing/worker-document/commits/develop), this is the only meaningful change though, and it is a new feature, so if we don't put it in, we have nothing to put in the worker-document release notes.

Normally, if there are only dependency updates, we change the release to a patch release and just have this as the release notes:

```
#### Version Number
${version-number}

#### New Features
- None

#### Bug Fixes
- Dependency updates only in this release.

#### Known Issues
- None
``` 

However, I'm not sure if this is right for the worker-document given the change above (i.e. the new env var is a new feature), so maybe we should include it in the worker-document release notes?

Reply:
There should be an entry under New Features as you say, to justify the minor version bump, but it could just say "Upgraded to new version of Worker Framework" or something like that if that makes more sense than repeating details.